_posts/2021-12-14-dec-community-call-summary.md

```diff
@@ -0,0 +1,113 @@
+If these highlights whet your interest,
+then the video is already up on YouTube for you to watch:  
+https://www.youtube.com/watch?v=0vdfNojvlck.
+We've also provided a link to notes from the discussion in the breakout rooms:  
+https://bit.ly/US-RSE-Dec-CC-Breakout.
+If you don't watch anything else, you can at least
+get a quick update on news from Ian Cosden,
+and check out responses to Chris Hill's
+Mentimeter poll asking attendees
+what else they'd like to get the
+newly udpate US-RSE logo on,
+after the
+[US-RSE 2021 Winter T-Shirt campaign](https://www.customink.com/fundraising/us-rse-2021?side=front&type=3&zoom=)
+comes to an end. (Spoiler alert: "Mugs" was a popular choice.)
+
+## Lightning talks
+
+After news and the Mentimeter poll,
+the call moved on to the discussion topic.
+David Nicholson got things started
+by talking about how recently there has been growing interest in code review for RSEs.
+He mentioned the  
+[Research Code Review Community](https://github.com/ResearchCodeReviewCommunity)
+and the Software Saved Institute's
+[Collaborations 2022 Workshop](https://esciencelab.org.uk/announcements/projects/2021/10/29/ssi-collaborations-workshop-2022/)
+where code reviews in research software will be one of the themes.
+Then David introduced two lightning talks
+that would help guide community discussion in the breakout rooms.
+
+The [first talk](https://youtu.be/0vdfNojvlck?t=645)
+from [Jeff Carver](http://carver.cs.ua.edu/)
+provided a crash course in constructive code reviews,
+answering a couple of key questions:
+* What are the some of the goals of doing code review?
+* How should I approach the review if I'm the one submitting code,
+  or if I'm reviewing someone else's code?
+
+In the [second talk](https://youtu.be/0vdfNojvlck),
+[Pat Schloss ](http://www.schlosslab.org/)
+introduced the idea of
+[Code Clubs](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1008119#sec009),
+which he developed in his research group
+to achieve many of the same goals of code review,
+such as knowledge sharing.
+
+Interestingly, both Jeff and Pat made points
+about how reading code, reviewing others' code,
+and getting feedback on how we write code
+can fill some of the gaps
+left in traditional computer science education,
+that often encourages students to learn by writing,
+not reading.
+David pointed out that Greg Wilson
+had shared in the US-RSE Slack the book
+["The Programmer's Brain"](https://www.manning.com/books/the-programmers-brain)
+that makes similar observations about how
+coding is taught,
+and describes methods for improving reading comprehension.
+
+## Breakout room discussion
+
+Then the attendees entered the breakout rooms to discuss.
+Each group took notes, and then
+came back to the main room at the end of the call
+[to report out](https://youtu.be/0vdfNojvlck?t=1575).
+One common idea across groups was
+that of automating as much as possible
+--e.g., with linters and continuous integration--
```

Comment:
change "--"s to commas

Reply:
What, you don't like en dashes? 🙂 

I'd rather keep this one because you end up with a lot of commas in this sentence otherwise.

Reply:
I think they should be em dashes if dashes. Also because of the line breaks, they have spaces on one side but not the other.  I could see "word -- word" or "word---word" or "word, word" but not "word --word" and "word-- word"

Reply:
For this one, I think commas would be fine. I think it reads just fine with commas.

Reply:
Thank you, didn't realize the line breaks were the cause. Fixed.

They are em dashes, my bad for not knowing the punctuation lingo.